tests/integration-tests/tests/arm_pl/test_arm_pl/test_arm_pl/pcluster.config.ini

```diff
@@ -0,0 +1,22 @@
+[global]
+cluster_template = default
+
+[aws]
+aws_region_name = {{ region }}
+
+[cluster default]
+base_os = {{ os }}
+key_name = {{ key_name }}
+vpc_settings = parallelcluster-vpc
+scheduler = {{ scheduler }}
+master_instance_type = {{ instance }}
+compute_instance_type = {{ instance }}
+initial_queue_size = 0
+max_queue_size = 2
+
+
+[vpc parallelcluster-vpc]
+vpc_id = {{ vpc_id }}
+master_subnet_id = {{ public_subnet_id }}
+compute_subnet_id = {{ private_subnet_id }}
+use_public_ips = false
```

Comment:
Why avoid use of public IPs?

Reply:
`use_public_ips = false` prevents allocating an EIP to master node. It was suggested by demartinofra to reduce EIPs needed in integration tests

Reply:
👍 thank you good sir